test/sql/join/asof/test_asof_join_missing.test

```diff
@@ -0,0 +1,144 @@
+# name: test/sql/join/asof/test_asof_join_missing.test
+# description: Test As-Of join with missing matches
+# group: [asof]
+
+statement ok
+PRAGMA enable_verification
+
+# These test stress several aspects of the matching:
+# * Probe inequality less than the minimum (no match)
+# * Probe equality missing (no match)
+# * More than 64 valid probe entries (mask => SV construction)
+# * First radix bin empty.
+# * First payload bin empty 
+# * Multiple scanned payload blocks
+
+# Change to true to check results
+statement ok 
+PRAGMA debug_asof_iejoin=False
```

Comment:
Perhaps we can run this in a `loop i 0 2` and set `debug_asof_iejoin ` to true for the second iteration to check that the results are the same in both instances?